react/react_events/README.md

```diff
@@ -77,7 +79,7 @@ Alright, let's look at this line-by-line. First, we're using object destructurin
 
 Then, we return our JSX. We've got a containing `div`, because our component can only render one element at a time. Inside that `div`, we have a `button` element with an **event listener**: `onClick`.
```

Comment:
`we have a button element with an event listener` -- is it accurate to say that the `onClick` prop is an event listener?